editor/scene/2d/tiles/tile_map_layer_editor.cpp

```diff
@@ -1114,8 +1114,7 @@ HashMap<Vector2i, TileMapCell> TileMapLayerEditorTilesPlugin::_draw_rect(Vector2
 	// Get or create the pattern.
 	Ref<TileMapPattern> pattern = p_erase ? erase_pattern : selection_pattern;
 
-	HashMap<Vector2i, TileMapCell> err_output;
-	ERR_FAIL_COND_V(pattern->is_empty(), err_output);
+	ERR_FAIL_COND_V(pattern->is_empty(), (HashMap<Vector2i, TileMapCell>()));
```

Comment:
Yes, this was a copy before. I assume this was done as a separate variable to avoid the problem of commas inside macros, but there is a solution: Wrap the macro arguments in parenthesis. So in this case, the value returned by this error macro is `(HashMap<Vector2i, TileMapCell>())`.